src/Paramore.Brighter.MessagingGateway.AWSSQS/SqsMessagePublisher.cs

```diff
@@ -42,7 +43,7 @@ public SqsMessagePublisher(string topicArn, AmazonSimpleNotificationServiceClien
             _client = client;
         }
 
-        public string Publish(Message message)
+        public async Task<string> Publish(Message message)
```

Comment:
As with the comment above about breaking changes, since this is a public method on a public class, we should be adding a new method for the async version, even if the sync version is just calling `GetAwaiter().GetResult()` on the new method.